en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,201 @@
+
+## Tuning (advanced)
+
+To get an overview of all parameters that are related to the ackermann rover module navigate to the _Rover Ackermann_ group in the _Parameters_ section of QGroundControl.
+
+### General Parameters
+
+These parameters affect the general behaviour of the rover. This will influence both auto and manual modes.
+
+- [RA_MAX_SPEED](../advanced_config/parameter_reference.md#RA_MAX_SPEED): This is the speed the rover drives at maximum throttle. It is used for a feed-forward term on the speed controller in mission mode and necessary for the [acceleration slew rate](#slew-rates).
+
+#### Slew Rates
+
+Slew rates limit how fast the signal that is sent to the motors is allowed to change:
+
+- [RA_MAX_ACCEL](../advanced_config/parameter_reference.md#RA_MAX_ACCEL): This is the limit of how fast the rover can accelerate (this is not based on acceleration measurements but on assumed linear relation between the throttle and the maximum rover speed).
+
+:::warning
+[RA_MAX_SPEED](../advanced_config/parameter_reference.md#RA_MAX_SPEED) needs to be set for this to work!
+:::
+
+- [RA_MAX_ACCEL](../advanced_config/parameter_reference.md#RA_MAX_ACCEL): This is the limit on the steering rate.
+
+### Mission Parameters
+
+These parameters only affect the mission flight mode.
+
+:::warning
+The parameters in [Tuning (basic)](#tuning-basic) have to be set to drive missions!
+:::
+
+#### Pure Pursuit Parameters
+
+The module uses a control algorithm called _Pure pursuit_ and it can be beneficial to understand how it works to properly tune (see [Pure pursuit algorithm](#pure-pursuit-algorithm)).
+The look ahead distance sets how aggressive the controller behaves and is defined as follows:
+
+$$
+l_d = v \cdot k
+$$
+
+It depends on the velocity $v$ of the rover and a tuning parameter $k$ that can be set with the parameter [RA_LOOKAHEAD_TUN](#RA_LOOKAHEAD_TUN).
+
+:::info
+A lower value of `RA_LOOKAHEAD_TUN` makes the controller more aggressive but can lead to oscillations!
+:::
+
+To deal with the edge case that the line segment is outside the look ahead radius around the rover there are 2 steps:
+
+1. The look ahead is scaled to the crosstrack error, which creates an intersection point for the rover to drive towards.
+2. Scaling the look ahead too much can have undesirable effects such as the rover driving a huge circle to return to the path, rather then taking a straight line. For this purpose the maximum look ahead radius can be capped with the parameter [RA_LOOKAHEAD_MAX](#RA_LOOKAHEAD_MAX), after which the rover will drive directly towards the current waypoint until it gets close enough to the line segment again.
+
+To summarize, the following parameters can be used to tune the controller:
+
+| Parameter                                                                                                   | Description                             | Unit |
```

Comment:
FYI, note that generally we use a table to summarise the params, just as you have done. This is prefixed with an anchor, with the same name and links to the actual parameter.
Then in the rest of the doc we link to the table entry - you can see that above for `RA_LOOKAHEAD_MAX`. You don't have to link everything to the table if you want, but you should link the first instance, and other instances should be code style to highlight them.

This ensures that people can get the info they want without heading out of the context, but at most they will have one jump.